ressources/call_for_contribution.txt

```diff
@@ -0,0 +1,37 @@
+Dear Coq users and developpers,
+
+We would like to shed some light on our new project "Coq Platform Doc".
+Our goal is to provide an online compilation of short, practical and interactive
+tutorials and how-to guides about Coq and the Coq Platform. We wish this content
+to be part of the next Rocq website. A detailed description of the project can
+be found in our Coq Enhancement Proposal (CEP) here:
+https://github.com/coq/ceps/pull/91
+
+The first few tutorials are available as well as  a (very preliminary) online
+interface here:
+https://www.theozimmermann.net/platform-docs/
```

Comment:
```suggestion
https://www.theozimmermann.net/platform-docs/
(note that this is a temporary URL that will cease to be available
once the tutorials are migrated to the official Rocq website)
```

Reply:
It is already going to change when moved to a more official repo, right ?

Reply:
Yes, but I was expecting the two events to be coordinated / happen at the same time.

Reply:
Yes, @tabareau said we could expect to have the new website before the end of the year. But indeed, if we want to move the repo before that, we should be able to.

So my suggestion could become:

```suggestion
https://www.theozimmermann.net/platform-docs/
(note that this is a temporary URL that will cease to be available
once the tutorials are migrated to the official Coq organization)
```